Documentation/design-docs/code-versioning.md

```diff
@@ -332,7 +332,8 @@ In order to do step 3 the CodeVersionManager relies on one of two different mech
 All methods have been classified to use at most one of the techniques, based on:
 
 ```
-MethodDesc::IsVersionableWithPrecode()
+MethodDesc::IsEligibleForTieredCompilation() && MethodDesc::IsTieredMethodVersionableWithPrecode()
```

Comment:
Updated to make vtable slot backpatching a function of code versioning rather than a function of tiering. Restored the other `IsVersionableWith*` functions and added an `IsVersionableWithoutJumpStamp()`, which I think would be useful in the future as well with possibly more versioning techniques.